src/App.jsx

```diff
@@ -23,9 +23,13 @@ class App extends React.Component {
   updateAppProductId(productId) {
     this.setState({
       productId: productId
+    }, () => {
+      window.scrollTo({
```

Comment:
I like the scrollTo method with this smooth option! 
I will use this for my review scroll button too.

Reply:
Are you passing a callback into setState? 🤯